src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -313,6 +317,19 @@ def _get_job_logs(self, req: JobRequest) -> None:
             )
             raise
 
+    def _split_request_by_job_id(self, rq: dict) -> list:
+        """
+        If request data comes with job_id_list instead of job_id, convert it into
+        JobRequest objects each having a single job_id
+        """
+        insts = []
+        for job_id in rq["content"]["data"]["job_id_list"]:
```

Comment:
This should probably check to see that `job_id_list` is a list, otherwise it'll crash.